README.md

```diff
@@ -10,6 +10,27 @@ Note that [RStudio Server Pro](https://www.rstudio.com/products/rstudio-server-p
 
 ## Installation
 
+### Pre-reqs
+#### Install [nbserverproxy](https://github.com/jupyterhub/nbserverproxy)
+```
+pip install git+https://github.com/jupyterhub/nbserverproxy
+```
+
+Either install the nbserverproxy extensions for the user:
+```
+jupyter serverextension enable --py nbserverproxy
+```
+
+Or install the nbserverproxy extensions for all users on the system:
+```
+jupyter serverextension enable --py --sys-prefix nbserverproxy
+```
+#### Install rstudio
+```
+conda install rstudio
```

Comment:
Thanks for the patch! :)

This looks great. Can you please also add that installing rstudio from distribution package is also an option?

Reply:
Sure!

Although I may be missing a step to get it to work... It didn't work out of the box, if I installed R Studio Server rpm instead of using `conda`

The rpm doesn't add `rsession` to the PATH which for testing purposes I solved with `export PATH=/usr/lib/rstudio-server/bin/:$PATH`

However, it's failing to create the rsession:
```
➜  ~ sudo ps -ef | grep rsession
vagrant   4639  4625  0 16:52 ?        00:00:00 [rsession] <defunct>
```

It looks like JupytreHub is throwing a 599 error: ` 599 GET /user/vagrant/proxy/33667/`

Thoughts?

Reply:
When debugging issues like this I typically have to invoke rsession with PATH and LD_LIBRARY_PATH set according to what the extension does, along with all of the CLI args.

I use the RStudio .deb packages on Ubuntu and need to set env vars similar to what is in this repository's Dockerfile.

That being said, we can probably move the rpm problem to a new issue and I can try to debug.